### PR TITLE
Updated readme per marketing feedback

### PR DESCRIPTION
**Pull request by crfroehlich**
https://app.clubhouse.io/ns8/story/29154/update-magento-integration-repo-to-adhere-to-public-repo-standards

https://www.dropbox.com/sh/miwynqqerfb8vx1/AAApNnZL7HHL_dKbD-3SmwU4a/NS8%20for%20Magento%202.paper?dl=0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
+The system requirements to use NS8 with Magento are:
 
 - Magento version 2.3.* (all 2.3 versions are supported)
 - PHP 7.1+
 
+Additionally, you must configure all of Magento’s background task operations to run automatically.
+
 ## Manual Install
 
 1. SSH to the Magento server and navigate to the root path of the Magento install, frequently in `/var/www/html`.
```
Comment (rob-ns8) [inline README.md:22]: Not sure how exact we need to be with the document in Dropbox, but here's this sentence in Dropbox:

>1. SSH to the Magento server and navigate to the root path of your Magento install, which is frequently located in `/var/www/html`, and run the following commands:

File: README.md [hunk 2]
```diff
@@ -25,10 +25,8 @@ The following items are requirements for using Protect with Magento:
...
 1. `$ sudo bin/magento module:status`
-1. Confirm that NS8 Protect is listed.
+1. After running the commands outlined in steps two through six, you can access NS8 through your Magento instance. 
 
 ## License
 
-See [License](./LICENSE)
-
- © [ns8inc](https://ns8.com)
+For more information about NS8, see our [License](./LICENSE) or visit us at [ns8inc](https://ns8.com).
```
Comment (rob-ns8) [inline README.md:32]: In the Dropbox document, the link text at the end of this sentence is `ns8.com` (not `ns8inc`).

File: README.md (hunk 1 above)
Comment (crfroehlich) [reply README.md]: Good catch. Fixed.

File: README.md (hunk 2 above)
Comment (crfroehlich) [reply README.md]: Good catch; fixed.

**Pull request merged**